src/cuga/backend/server/main.py

```
@@ -2470,10 +2471,27 @@ async def get_agents_list(current_user: Optional[UserInfo] = Depends(require_man
+        try:
+            from cuga.backend.server.config_store import load_config
+
+            config, _ = await load_config(None, "cuga-default")
+            if config and isinstance(config.get("agent"), dict):
+                ag = config["agent"]
+                if isinstance(ag.get("name"), str) and ag["name"].strip():
+                    name = ag["name"].strip()
+                if isinstance(ag.get("description"), str) and ag["description"].strip():
+                    description = ag["description"].strip()
+        except Exception:
+            pass
```

Ruff (0.15.6)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)